tests/pylorax/blueprints/example-http-server.toml

```diff
@@ -20,7 +20,7 @@ version = "7.2.*"
 
 [[packages]]
 name = "tmux"
-version = "2.8"
+version = "*"
```

Comment:
You will also need to change all the places where we look for 2.8 in the test results. eg. tests/pylorax/test_server.py,

Reply:
Well, turns out not all of them but only 2. The rest will be fine as long as Fedora has version 2.8.